src/test/scala/edu/ie3/simona/event/listener/RuntimeEventListenerSpec.scala

```diff
@@ -102,8 +102,12 @@ class RuntimeEventListenerSpec
         )
       }
 
-      def roundDuration(dur: Double): Double =
-        roundAt(5)(dur / 1000)
+      def convertDuration(duration: Double): String = {
+        val hh = ((duration / 1000 / 60) % 24).toInt
+        val mm = ((duration / 1000 / 60) % 60).toInt
+        val ss = roundAt(5)(duration / 1000) % 60
+        s"$hh h: $mm m: $ss s"
+      }
```

Comment:
I'd advise against having the same function a second time, because that would need to be maintained as well. Maybe rather test against a set amound of milliseconds, so you'd test against something like `[...] Total runtime: 4 h: 19 m: 3 s`